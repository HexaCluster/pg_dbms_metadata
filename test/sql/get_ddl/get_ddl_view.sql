SELECT dbms_metadata.get_ddl('VIEW','gg_d','gdmmm');

SELECT dbms_metadata.get_ddl('VIEW','Global fines','gdmmm');

-- tests for schema as NULL
SET search_path TO public, gdmmm;

SELECT dbms_metadata.get_ddl('VIEW','gg_d');

SELECT dbms_metadata.get_ddl('VIEW','Global fines');
