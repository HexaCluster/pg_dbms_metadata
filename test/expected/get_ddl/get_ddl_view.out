                    get_ddl                    
-----------------------------------------------
 CREATE VIEW gdmmm.gg_d AS  SELECT table_all.a+
    FROM gdmmm.table_all
(1 row)

                                        get_ddl                                         
----------------------------------------------------------------------------------------
 CREATE VIEW gdmmm."Global fines" AS  SELECT "Sample unlogged table"."name of customer"+
    FROM gdmmm."Sample unlogged table"
(1 row)

SET
                    get_ddl                    
-----------------------------------------------
 CREATE VIEW gdmmm.gg_d AS  SELECT table_all.a+
    FROM table_all
(1 row)

                                        get_ddl                                         
----------------------------------------------------------------------------------------
 CREATE VIEW gdmmm."Global fines" AS  SELECT "Sample unlogged table"."name of customer"+
    FROM "Sample unlogged table"
(1 row)

